packages/govtnz-ds-website/src/docs/components/Checkboxes.md

```diff
@@ -5,6 +5,7 @@
 Interact with each example to see how it works.
 
 <ExampleContainer>
+    <ExampleHeading></ExampleHeading>
```

Comment:
This means we will need an empty `<ExampleHeading></ExampleHeading>` in the markdown files so it knows that there is no child.  I initially added the Example default text in here but it didn't seem to work that well with the `ExampleHeading.tsx`  component.